### PR TITLE
fix: [ANDROAPP-2701] Dial fab with RTL support

### PR DESCRIPTION
**Pull request by Balcan**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/ANDROAPP-2701)

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/androidTest/java/org/dhis2/common/viewactions/CommonViewActions.kt [hunk 1]
```diff
@@ -130,7 +129,7 @@ fun scrollToPositionRecyclerview(position: Int): ViewAction {
     }
 }
 
-fun clickOnFabChild(): ViewAction {
+/*fun clickOnFabChild(): ViewAction {
```
Comment (JaimeToca) [inline app/src/androidTest/java/org/dhis2/common/viewactions/CommonViewActions.kt:132]: ?

File: app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt [hunk 2]
```diff
@@ -347,7 +347,9 @@ class TeiDashboardTest : BaseTest() {
 
         relationshipRobot {
             clickOnFabAdd()
+            waitToDebounce(500)
             clickOnRelationshipType()
+            waitToDebounce(500)
```
Comment (JaimeToca) [inline app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt:352]: not sure if you need this? Animations are disabled in Espresso. Did you try removing them?

**Review by ferdyrod — APPROVED** [review]

File: app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt (hunk 2 above)
Comment (Balcan) [reply app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt:352]: I tried, but could not make it work without this

Comment (JaimeToca) [reply app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt:352]: in your local machine or bitrise?

**Pull request closed without merging**